spec/cli_spec.rb

```diff
@@ -4,7 +4,35 @@
   context '#run_suite' do
     it 'calls spec runner' do
       expect_any_instance_of(FailSpell::SpecRunner).to receive(:run_and_store_results)
-      FailSpell::CLI.new.run_suite
+      described_class.new.run_suite
+    end
+  end
+
+  context '#rerun_failed' do
+    it 'updated successful tests in json file' do
+      copy_file_path = '/tmp/test_rerun_failed_with_success.json'
+      failed_test_path = './spec/lib/relateiq/list_spec.rb'
+      failed_test_line_number = 78
+      full_failed_spec_path = "#{failed_test_path}:#{failed_test_line_number}"
+      FileUtils.cp('./spec/fixtures/result.json', copy_file_path)
+
+      show_block = Proc.new do |failed_specs, &block|
```

Comment:
Unused block argument - `failed_specs`. If it's necessary, use `_` or `_failed_specs` as an argument name to indicate that it won't be used.<br>Use `proc` instead of `Proc.new`.